website/docs/installation.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Installation
+---
+
+Kaito requires an existing Kubernetes cluster. This guide provides steps for common cloud providers, but the cluster can be hosted on any cloud provider.
+
+## Prerequisites
+
+Before you begin, ensure you have the following tools installed.
+
+- [Helm](https://helm.sh) to install this operator
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) to view Kubernetes resources
+- [git](https://git-scm.com/downloads) to clone this repo locally
+- [yq](https://github.com/mikefarah/yq) to process yaml files
+- [jq](https://jqlang.github.io/jq/download) to process JSON file
+
+**Important Note**:
+Ensure you use a release branch of the repository for a stable version of the installation.
+
+For each provider, these are additional prerequisites:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="provider">
+<TabItem value="azure" label="Azure" default>
+
+- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli) to provision Azure resources
```

Comment:
we should not have any cloud provider here. you can think of it as installing on a kind cluster. however, in the cloud providers section we can have cloud provider specific instructions for supporting provisioning etc for specific cloud providers